### ISSUE TITLE
[request] Screen filters for ps vita port.

### ISSUE THREAD

**Issue by shim80** [mgba-emu/mgba#448]
Do you can add screen filters (HQ2X, linear etc...) option for the vita port of mGBA?

**Comment by endrift** [comment]
I'll need support for this from @xerpi's vita2d lib before I can actually do it.

**Comment by xerpi** [comment]
I guess that requires a custom Fragment/Pixel shader?

**Comment by endrift** [comment]
Yep, exactly.

**Comment by xerpi** [comment]
I think that it would be better to just create your own GXM wrapper instead of using vita2d, so you will have a lot more control over the GPU. You can base it on the vita2d, I have no problem with that.
If I had more free time I would be glad to write it myself.

**Comment by endrift** [comment]
Yeah I guess I could fork vita2d and drop the stuff I don't want (which is a fair bit), but there's still a fair bit of stuff I DO want to use.

Also like. I'm in no rush. This probably won't happen before December, and likely won't happen this year.

**Comment by shim80** [comment]
Hi.
Whats'up?

**Comment by xerpi** [comment]
I added some GXM samples to vitasdk's samples a few months ago, they might be useful: https://github.com/vitasdk/samples/tree/master/gxm
